platforms/hanwha-wn/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+include(ExternalProject)
+include(ProcessorCount)
+include(../shared/shared.cmake)
+
+set(BOOST_BUILD_PATH ${CMAKE_CURRENT_BINARY_DIR}/boost-prefix/src/boost)
+
+file(WRITE ${BOOST_BUILD_PATH}/user-config.jam "using gcc : arm : \"$ENV{WN}/arm-linux-gnueabi-g++\"")
+
+set (BOOST_ARCH_NAME boost_1_63_0.tar.bz2)
```

Comment:
These build files are for building Connect SDK. Code and build files for hanwha integration app will be in a separate private repo, and will use boost libraries built here